dune_daq_services/runservices/runnumber-rest/runnumber-rest.yaml

```diff
@@ -8,7 +8,6 @@ metadata:
   name: runnumber-rest
   namespace: {{ DUNE_runservices.namespace }}
 spec:
-  minReadySeconds: 5
```

Comment:
Any reason this is being removed?

Reply:
I think I may have removed it by mistake, what does it do?

Reply:
It requires the pod to wait 5 seconds before being considered up and running.  This in theory helps slow the respawn rate when postgresql isn't up yet.

Reply:
I'll add it again